images/nginx-slim/build.sh

```diff
@@ -47,6 +49,12 @@ get_src()
 mkdir "$BUILD_PATH"
 cd "$BUILD_PATH"
 
+if [[ ${ARCH} == "ppc64le" ]]; then
+  apt-get update && apt-get install -y software-properties-common && \
+    add-apt-repository -y ppa:ibmpackages/luajit
```

Comment:
Will not make any difference, I intentionally divided them into two set of commands to increase readability.